src/main/java/personList/ListActions.java

```diff
@@ -0,0 +1,70 @@
+package personList;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class ListActions {
+    public static boolean contains(List<Person> personList, Person person) {
+        boolean containsResult = false;
+        PersonsComparator comparePersons = new PersonsComparator();
+        for (Person e : personList) {
+            if (comparePersons.compare(e, person) == 0) {
+                containsResult = true;
+            }
+        }
+        return containsResult;
+    }
+
+    public static boolean equalLists(List<Person> personList1, List<Person> personList2) {
+        Iterator<Person> iter = personList1.iterator();
+        boolean containsResult = false;
+        if (personList1.size() != personList2.size()) {
+            containsResult = false;
```

Comment:
As above, you can instantly return false here, so we don't need an `else` statement

Reply:
Added instant return. But else statement I need here to compare objects in the list. First I compare sizes of lists and if they are equal - compare objects inside.